Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.job.num_failed_tasks.count,rate,,task,,The number of times the number of failed tasks in the application's jobs was sampled,0,spark,jb nm fld tsk cnt
+spark.job.num_active_stages.max,gauge,,,,The max number of active stages in the application's jobs,0,spark,jb act stg mx
+spark.job.num_active_stages.avg,gauge,,,,The average number of active stages in the application's jobs,0,spark,jb act stg avg
+spark.job.num_active_stages.median,gauge,,,,The median number of active stages in the application's jobs,0,spark,jb act stg med
+spark.job.num_active_stages.95percentile,gauge,,,,The 95th percentile number of active stages in the application's jobs,0,spark,jb act stg 95
+spark.job.num_active_stages.count,rate,,,,The number of times the number of active stages in the application's jobs was sampled,0,spark,jb act stg cnt
+spark.job.num_completed_stages.max,gauge,,,,The max number of completed stages in the application's jobs,0,spark,jb nm cmp stg mx
+spark.job.num_completed_stages.avg,gauge,,,,The average number of completed stages in the application's jobs,0,spark,jb nm cmp stg avg
+spark.job.num_completed_stages.median,gauge,,,,The median number of completed stages in the application's jobs,0,spark,jb nm cmp stg med
+spark.job.num_completed_stages.95percentile,gauge,,,,The 95th percentile number of completed stages in the application's jobs,0,spark,jb nm cmp stg 95
+spark.job.num_completed_stages.count,rate,,,,The number of times the number of completed stages in the application's jobs was sampled,0,spark,jb nm cmp stg cnt
+spark.job.num_skipped_stages.max,gauge,,,,The max number of skipped stages in the application's jobs,0,spark,jb nm skp stg mx
+spark.job.num_skipped_stages.avg,gauge,,,,The average number of skipped stages in the application's jobs,0,spark,jb nm skp stg avg
+spark.job.num_skipped_stages.median,gauge,,,,The median number of skipped stages in the application's jobs,0,spark,jb nm skp stg med
+spark.job.num_skipped_stages.95percentile,gauge,,,,The 95th percentile number of skipped stages in the application's jobs,0,spark,jb nm skp stg 95
+spark.job.num_skipped_stages.count,rate,,,,The number of times the number of skipped stages in the application's jobs was sampled,0,spark,jb nm skp stg cnt
+spark.job.num_failed_stages.max,gauge,,,,The max number of failed stages in the application's jobs,0,spark,jb nm fld stg mx
+spark.job.num_failed_stages.avg,gauge,,,,The average number of failed stages in the application's jobs,0,spark,jb nm fld stg avg
+spark.job.num_failed_stages.median,gauge,,,,The median number of failed stages in the application's jobs,0,spark,jb nm fld stg med
+spark.job.num_failed_stages.95percentile,gauge,,,,The 95th percentile number of failed stages in the application's jobs,0,spark,jb nm fld stg 95
+spark.job.num_failed_stages.count,rate,,,,The number of times the number of failed stages in the application's jobs was sampled,0,spark,jb nm fld stg cnt

```

Comment:
a nitpick: I'd change all the "in the application's jobs" to "in the application", I think it's redundant because a spark app only has jobs and no other entities.